### ISSUE TITLE
Node 0.1.30 and getRequestPromise

### ISSUE THREAD

**Issue by ncr** [ciaranj/flickrnode#3]
I'm in the process of making flickr-spy work with current node.js:

```
ncr:~$ node -v
v0.1.30-23-g2b91f8d

TypeError: Object #<a Request> has no method 'getRequestPromise'
    at Urls.lookupUser (/Users/ncr/dev/flickr_spy/vendor/flickr/lib/urls.js:6:26)
```

**Comment by ciaranj** [comment]
I _think_ you need to update your submodule, as I _thought_ I'd fixed all the 0.1.30 issues, unfortunately this has meant a small change in the APIs :(

**Comment by ciaranj** [comment]
I'm sorry, I missed that one, will fix now ! :)

**Comment by ciaranj** [comment]
Fixed in cd3ee36708a6bbff8597a1d67d56585d6eb47452

**Comment by ncr** [comment]
Thanks a lot Ciaran :)
